src/lib/components/auth/Auth0Button.svelte

```
@@ -1,12 +1,28 @@
 <script lang="ts">
+  import { onMount } from 'svelte';
```

ESLint

[error] 'onMount' is defined but never used.

(@typescript-eslint/no-unused-vars)